### DISCUSSION TITLE
Hooks not working correctly

### DISCUSSION THREAD

**Discussion by jcharnley** [directus/directus#2692] [Archive]
Does anyone see why this is not firing. its in a php file within /hooks in the docker container 

<img width="1012" alt="Screenshot 2020-04-30 at 12 21 24" src="https://user-images.githubusercontent.com/25176118/80704879-55cb1700-8add-11ea-8d79-2d42317023b4.png">

even if i hard code it with the example in the docs, it goes beserk  and keeps firing like crazy

<img width="870" alt="Screenshot 2020-04-30 at 12 24 55" src="https://user-images.githubusercontent.com/25176118/80705111-c1ad7f80-8add-11ea-8a89-de609be9998f.png">

**Comment by rijkvanzanten** [comment]
@philleepflorence any thoughts? 

@WoLfulus could it be something Docker specific?

**Reply by philleepflorence** [reply]
Not sure what could be happening, but to debug, create a plain hook file and error log the`func_get_args` to see if the hook is not firing or if there is an error preventing the hook from firing.

**Comment by jcharnley** [comment]
ok so i fixed it. I need to if statment when any directus triggers ran

```php
'item.update' => function ($collectionName, array $data) {
  if (strpos($collectionName, 'directus_') === false) {
    $client = new \GuzzleHttp\Client([
      'base_uri' => "hosturl/project/build-deploy?collection=$collectionName&id=" . $data['id']
    ]);
    try {
      $response = $client->request('GET', '');
    } catch (\Exception $e) {
    }
  }
},
```
